ui/lib/core/addon/components/search-select.js

```diff
@@ -42,6 +47,9 @@ export default Component.extend({
   shouldRenderName: false,
   init() {
     this._super(...arguments);
+    if (this.initialSelected) {
+      return this.set('selectedOptions', this.initialSelected); // need array helper to pass in.
```

Comment:
It will override any input values on component init, but this brings up a good point that we should write tests around our assumptions of how this works 👍

Reply:
Not as far as we can tell, but this is a good reminder to write tests around this functionality 👍

Reply:
After looking into this I think we need to refactor what we pass into the component from `form-field`, not necessarily update the component itself. Working on those changes now 😄

Reply:
I reverted this change after finding that the Search Select component is supposed to work how we wanted it to here (showing the initial values) -- the core of the issue is that our template/templates model is dirty. This is addressed in [this PR](https://github.com/hashicorp/vault/pull/9838) but until it's merged, the edit form for transformations is broken